build-tests/rush-stack-compiler-2.4-library-test/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "rush-stack-compiler-2.4-library-test",
```

Comment:
>ush-stack-compiler-2.4-library-test" [](start = 12, length = 36)

This seems like a lot of copy+pasting.  Is there any easy way to create a single project that tests all the compiler versions?

Reply:
Discussed offline and fixed.

---
In reply to: [234095426](https://github.com/Microsoft/web-build-tools/pull/947#discussion_r234095426) [](ancestors = 234095426)